src/preact/component/3p-frame.js

```diff
@@ -60,8 +60,12 @@ const DEFAULT_SANDBOX =
  */
 function ProxyIframeEmbedWithRef(
   {
+    allow = BLOCK_SYNC_XHR,
```

Comment:
Can we assert that any passed in `allow` contains `sync-xhr 'none'`?  It's an extremely important performance guarantee, and there is no acceptable reason to need the feature.

I'm worried we'll provide an override, forget to ban sync-xhr, and the included iframe will seriously hurt page performance.

Reply:
Done, thanks for the suggestion.